src/index.html

```diff
@@ -4,8 +4,8 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <link rel="stylesheet" href="../html/style.css" />
```

Comment:
Why did you change the path? Do you have a folder called 'html'?

Reply:
You shouldn't change our build environment setting, we have internal library that convert sass file to css file on build.
 
This is one of the major reasons for the poor code convention and pull request failure.